crates/bevy_color/src/srgba.rs

```diff
@@ -0,0 +1,462 @@
+use crate::color_difference::EuclideanDistance;
+use crate::oklaba::Oklaba;
+use crate::{Alpha, Hsla, LinearRgba, Luminance, Mix};
+use bevy_math::Vec4;
+use bevy_reflect::{Reflect, ReflectDeserialize, ReflectSerialize};
+use bevy_render::color::{Color, HexColorError, HslRepresentation, SrgbColorSpace};
+use serde::{Deserialize, Serialize};
+
+/// Non-linear standard RGB with alpha.
+#[derive(Debug, Clone, Copy, PartialEq, Serialize, Deserialize, Reflect)]
+#[reflect(PartialEq, Serialize, Deserialize)]
+pub struct Srgba {
+    /// The red channel. [0.0, 1.0]
+    pub red: f32,
+    /// The green channel. [0.0, 1.0]
+    pub green: f32,
+    /// The blue channel. [0.0, 1.0]
+    pub blue: f32,
+    /// The alpha channel. [0.0, 1.0]
+    pub alpha: f32,
+}
+
+impl Srgba {
```

Comment:
I think it would be good to include a link to where these definitions come from? E.g., [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/named-color)

Reply:
I copied them from the bevy Color class. The colors are a subset of the ones listed on MDN.

Having these is handy for debugging, but I don't know whether we should spend too much effort to ensure that the set of colors matches some external standard.

Reply:
I've edited the list of colors so that it now only includes the 16 VGA colors + NONE + DARK_GRAY.